airflow/providers/google/suite/hooks/drive.py

```diff
@@ -65,9 +65,9 @@ def __init__(
 
     def get_conn(self) -> Any:
         """
-        Retrieves the connection to Google Drive.
-
-        :return: Google Drive services object.
+                Retrieves the connection to Google Drive.
```

Comment:
This indentation causes doc build to fail :)

Reply:
Aha! Was trying to figure out why doc build is failing. Will make the changes 
Thanks @potiuk :)